ieeeconf/footer.tex

```diff
@@ -0,0 +1,31 @@
+% Bibliography
+
+\ifx\bibliocommand\undefined
+\else
+\printbibliography
+\fi
+
+\appendices
```

Comment:
Also, should this command only fire if `\myappendix` is defined? I'm not sure what all the side effects would be, but it may be cleaner to only execute this command when appendices are included